packages/office-ui-fabric-react/src/components/DetailsList/DetailsRowFields.tsx

```diff
@@ -53,41 +59,11 @@ export class DetailsRowFields extends BaseComponent<IDetailsRowFieldsProps, IDet
               data-automationid="DetailsRowCell"
               data-automation-key={column.key}
             >
-              {cellContent[columnIndex]}
+              {cellContentsRender}
             </div>
           );
         })}
       </div>
     );
   }
-
-  private _getState(props: IDetailsRowFieldsProps): IDetailsRowFieldsState {
-    const { item, itemIndex, onRenderItemColumn, shimmer } = props;
-
-    return {
-      cellContent: props.columns.map(column => {
-        let cellContent;
-
-        try {
-          const render = column.onRender || onRenderItemColumn;
-
-          cellContent = render && !shimmer ? render(item, itemIndex, column) : this._getCellText(item, column);
-        } catch (e) {
```

Comment:
Wasn't the try/catch here for cases when neither `column.onRender` or `onRenderItemColumn` are provided? If it's `undefined` you might get an error `undefined is not a function` in this edge case.

Reply:
From the history, it looks like it's there from the very beginning when David created it.

Reply:
Indeed, that's why I'm looking for if anyone has context. It seems like we'd only have an issue if the provided render function throws. Not sure if that's something worth guarding against.

Reply:
This was written in the time of react 0.14.

Reply:
I think it's ok to remove.

Reply:
Thank you for providing context :)